bash_scripts/cron.jobs

```diff
@@ -6,7 +6,8 @@ LOG_PATH  = /home/jaleazo/projects/Spotify-2.0/logs
 
 * * * * *  $PROJ_PATH/Proj.sh Playback_Tracker  >  $LOG_PATH/Playback-Tracker.log    2>&1    # Every Minute
 0 3 * * *  $PROJ_PATH/Proj.sh Backup_Data       >> $LOG_PATH/Backup-Data.log         2>&1    # Every Day at 3 AM
-0 2 * * 1  $PROJ_PATH/Proj.sh Weekly_Report     >> $LOG_PATH/Weekly-Report.log       2>&1    # Every Monday at 2AM
```

Comment:
I didn't catch this on the previous PR, but do these properly redirect stderr into the log file? I would expect the command to be `$PROJ_PATH/Proj.sh Weekly_Report 2>&1 >> $LOG_PATH/Weekly-Report.log` rather than the `2>&1` at the end

Reply:
The 2>&1 is more of an 'option' to '>>' so we are just saying '>>' push all output to the file and 2>&1 just defines what "stuff" we want piped ie stderr and stdout